BrainPortal/cbrain_plugins/cbrain-plugins-base/cbrain_task/start_vm/common/start_vm.rb

```diff
@@ -0,0 +1,60 @@
+
+#
+# CBRAIN Project
+#
+# Copyright (C) 2008-2012
+# The Royal Institution for the Advancement of Learning
+# McGill University
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.  
+#
+
+class CbrainTask::StartVM #:nodoc:
+  
+  def param_validation_errors #:nodoc:
+    errors = Hash.new
```

Comment:
Ah I see... well explaining that in a short paragraph just above the method would have helped. No need to make a new commit for that though.